Code/Framework/AzFramework/Platform/Windows/AzFramework/Windowing/NativeWindow_Windows.cpp

```diff
@@ -193,10 +193,17 @@ namespace AzFramework
 
     HWND NativeWindowImpl_Win32::GetWindowedPriority()
     {
+        bool isNotTopMost = AZ::Debug::Trace::Instance().IsDebuggerPresent();
+
+        // If we are launching with pix enabled we are probably trying to do a gpu capture and hence need to be able to
+        // access RenderDoc or Pix behind the O3de app.
+#if defined(USE_PIX)
+        isNotTopMost = true;
+#endif
         // If a debugger is attached and we're running in Windowed mode instead of Fullscreen mode,
         // don't make the window TOPMOST. Otherwise, the window will stay on top of the debugger window
         // at every breakpoint, crash, etc, making it extremely difficult to debug when working on a single monitor system.
-        return AZ::Debug::Trace::Instance().IsDebuggerPresent() ? HWND_NOTOPMOST : HWND_TOPMOST;
+        return isNotTopMost ? HWND_NOTOPMOST : HWND_TOPMOST;
```

Comment:
Suggestion: Consider flipping the logic and calling it "allowTopMost". I've always found positive logic and variable names a bit easier to read than negative logic.